DB/schema.sql

```diff
@@ -17,7 +17,7 @@ CREATE TABLE author (
 
 CREATE TABLE book (
 	id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-	publisher VARCHAR(120),
+	publisher VARCHAR(125),
 	cover_state VARCHAR(120),
```

Comment:
Nice work done!
It would be nice to use the same indentation size for all codes